src/OrchardCore.Modules/OrchardCore.Spatial/Drivers/CoordinateFieldDriver.cs

```diff
@@ -0,0 +1,44 @@
+using System.Threading.Tasks;
+using OrchardCore.ContentManagement.Display.ContentDisplay;
+using OrchardCore.ContentManagement.Display.Models;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Spatial.Fields;
+using OrchardCore.Spatial.ViewModels;
+
+namespace OrchardCore.Spatial.Drivers
+{
+    public class GeoPointFieldDisplayDriver : ContentFieldDisplayDriver<GeoPointField>
+    {
+        public override IDisplayResult Display(GeoPointField field, BuildFieldDisplayContext context)
+        {
+            return Initialize<DisplayGeoPointFieldViewModel>("GeoPointField", model =>
```

Comment:
`GetDisplayType`, see other drivers (might be another name), this is used for editors.

Reply:
Thanks. The code was way out of date.